database/src/migrations/20230719141800_project_type_to_program.ts

```diff
@@ -0,0 +1,69 @@
+import { Knex } from 'knex';
+
+/**
+ * // add new table for region lookup and connection table to project and survey
+ *
+ * @export
+ * @param {Knex} knex
+ * @return {*}  {Promise<void>}
+ */
+export async function up(knex: Knex): Promise<void> {
+  await knex.raw(`
+	set search_path= biohub, public;
+
+
+    CREATE TABLE program (
+      program_id              integer                 GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1),
+      name                    varchar(50),
+      description             varchar(50),
+      record_effective_date   date,
+      record_end_date         date,
```

Comment:
Will need to add that 'unique end-date constraint' for this table. Examples in that readme in my PR.

Would be on (name, record_end_date) so that you can't have 2 entries with the same name (unless the record_end_date is not null)

Reply:
added